docs/events-api.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[events-api]]
+== Events API
+
+Agents capture different types of information, known as events.
+These events are sent to a single endpoint which then sorts and processes the events.
+Events can be:
+
+* Transactions
+* Spans
+* Errors
+* Metricsets
+
+You can learn more about events in the {apm-get-started-ref}/apm-data-model.html[APM Data Model] documentation. 
+
+[[events-api-endpoint]]
+[float]
+=== Endpoint
+Send an `HTTP POST` request to the APM Server `intake/v2/events` endpoint:
+
+[source,bash]
+------------------------------------------------------------
+http(s)://{hostname}:{port}/intake/v2/events
+------------------------------------------------------------
+
+For <<rum, RUM>> send an `HTTP POST` request to the APM Server `intake/v2/rum/events` endpoint instead:
+
+[source,bash]
+------------------------------------------------------------
+http(s)://{hostname}:{port}/intake/v2/rum/events
+------------------------------------------------------------
+
+[[events-api-response]]
+[float]
+=== Response
+
+On success, the server will respond with a 202 Accepted status code and no body.
+
+[[events-api-errors]]
+[float]
+=== Errors
+
+Events are processed one after the either.
```

Comment:
Typo (or do I not know this expression)?

Reply:
Haha, whoops. Typo!